ndcube/cube_utils.py

```diff
@@ -182,10 +178,13 @@ def index_sequence_as_cube(cubesequence, item):
     item_tuple = tuple(item_list)
     if item is None or (isinstance(item, tuple) and None in item):
         raise IndexError("None indices not supported")
-    try:
-        return get_cube_from_sequence(cubesequence, item_tuple, type_slice='sequence_as_cube')
-    except:
-        return get_cube_from_sequence(cubesequence, item_tuple)
+
+    # if the slicing is done across one cube. Then we can pass the item_tuple as
+    # (int, rest of the information) not (int, int , rest of the information)
+    if isinstance(item_tuple[0], int) and isinstance(item_tuple[1], int):
+        if item_tuple[0] == item_tuple[1]:
```

Comment:
I did some checking manual testing and the above condition (```if item_tuple[0] == item_tuple[1]:```) needs to be removed. 

At the moment, if ```item_tuple[0] == item_tuple[1]:```, a single slice of an (N-1)DCube is returned and a new ```NDCube``` object.  But if ```item_tuple[0] != item_tuple[1]``` a single slice of an (N-1)DCube is returned wrapped up in an ```NDCubeSequence```.

I'll merge this PR and then remove it as part of my current PR.